### DISCUSSION TITLE
I need help adding multiple usernames and passwords to my fork.

### DISCUSSION THREAD

**Discussion by ghost** [UseInterstellar/Interstellar#613] [Q&A] [answered]
Can someone please help me make it so that I can set up multiple usernames and passwords on my fork?

**Comment by 0mhx** [comment, marked as answer]
It's obvious. Duplicate the username/password line. 

Also please note that GitHub is not for deployment support, it is only for reporting issues in the CODE. 

Join the [discord server](https://discord.gg/unblock) for deployment support.

**Reply by ghost** [reply]
Sorry.

**Comment by replitidk** [comment]
> Can someone please help me make it so that I can set up multiple usernames and passwords on my fork?

 admindanke: '6895', ryan: '1391', bishop: '3912', adminvince: '1937', fortnitekid997: '9102' , royalxcypress: '4592' , guest: '1242' this is what it would be like this a example.
